test/integration/generated/appconfiguration/package.json

```diff
@@ -77,7 +77,8 @@
     "unit-test:browser": "echo skipped",
     "integration-test": "echo skipped",
     "integration-test:node": "echo skipped",
-    "integration-test:browser": "echo skipped"
+    "integration-test:browser": "echo skipped",
+    "docs": "echo skipped"
```

Comment:
Is adding "docs": "echo skipped" an intentional change or a merge conflict resolution problem?